packages/use-hooks/src/useColorMode/useColorMode.ts

```diff
@@ -14,12 +14,14 @@ const colorModeSchema = z
  * A composable hook that provides color mode (dark/light) functionality.
  */
 export function useColorMode(opts: UseColorModeOptions = {}) {
-  const { initialColorMode = 'dark', overrideColorMode } = opts
+  const { initialColorMode = 'system', overrideColorMode } = opts
 
   /** Toggles the color mode between light and dark. */
   function toggleColorMode() {
+    const darkLightMode =
```

Comment:
Updated this part so when you click toggle it switches to the opposite of your system colorMode if colorMode is 'system'